src/Services/Terminal/Terminal.js

```diff
@@ -223,6 +231,10 @@ module.exports = function (settings) {
           terminalState: TERMINAL_STATE_READY,
         });
 
+        if (UNEXPECTED_TERMINAL_ERRORS.some(e => response.includes(e))) {
+          reject(new TerminalRuntimeError.TerminalUnexpectedError(response));
```

Comment:
And please add return after reject, otherwise, resolve will be execute

Reply:
There is also a bug in Sonar: https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=622&id=Travelport-Ukraine_uapi-json&open=AYc3wGnDpB0WZnAf-lNf

I think we should just rewrite `executeCommand` to be async function instead of sync function returning Promise